### ISSUE TITLE
Kommunikation

### ISSUE THREAD

**Issue by MalteStorm** [MalteStorm/Test#5]
Organisation und FileSharing lassen sich wunderbar über GitHub lösen. Aber was ist der beste Weg, um eine schnelle, jeden erreichende und einfache Komunikation zu ermöglichen?

**Comment by MalteStorm** [comment]
Habe mit Lars, dem Oraginteressiertem telefonieret. Seine Ideen:
Skype für Kommunikation mit Leuten, die nicht in Ac sind.
Hirachie einführen: Unterteams mit Teamleiter, der Austausch mit anderen Teams hält

**Comment by MalteStorm** [comment]
Es gibt keine pm funktion oder? Was ist die beste Alternative dazu?

**Comment by fdietze** [comment]
Nein, gibt es nicht. Da muss auf ein alternatives Medium zurückgegriffen
werden.

**Comment by MalteStorm** [comment]
> Nein, gibt es nicht. Da muss auf ein alternatives Medium zurückgegriffen werden.
> das hier ist ne zweite Zeile

Wie schaffe ich es, dass die Kopfzeile mitgeschrieben wird?
via emailreplay?
